feature-group-definitions/image-set.yml

```diff
@@ -2,3 +2,21 @@ name: image-set()
 description: The `image-set()` CSS function provides a set of images at different resolutions or pixel densities, which the browser can pick from, depending on the device capabilities.
 spec: https://drafts.csswg.org/css-images-4/#image-set-notation
 caniuse: css-image-set
+# TODO: Remove this when these BCD PRs have landed:
+# https://github.com/mdn/browser-compat-data/pull/22902
+# https://github.com/mdn/browser-compat-data/pull/22912
+status:
+  baseline: low
+  baseline_low_date: 2023-05-05
+  support:
+    chrome: "113"
+    chrome_android: "113"
+    edge: "113"
+    firefox: "89"
+    firefox_android: "89"
+    safari: "14"
+    safari_ios: "14"
```

Comment:
caniuse says this is partial until 17. I see you did not reflect that change in the BCD PR. For posterity, want to make a note about what you're thinking here?

(marking approved because I'm presuming reasonableness and I mostly want something to look back on in case anyone asks)

Reply:
Oh, looks like more BCD updates will be needed here to be consistent between Firefox and Safari regarding support for `type()`.

Reply:
@ddbeck can you review https://github.com/mdn/browser-compat-data/pull/22902 and https://github.com/mdn/browser-compat-data/pull/22912? I'll need to build on those.

Reply:
@foolip Approved and merged the Chrome/Safari PR, but I had questions on the Firefox one. Thank you!

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/22939 and will change to Safari 17 here.